griptape/drivers/memory/conversation/griptape_cloud_conversation_memory_driver.py

```diff
@@ -128,7 +128,7 @@ def load(self) -> tuple[list[Run], dict[str, Any]]:
 
         runs = [
             Run(
-                id=m["metadata"].pop("run_id"),
+                id=m["metadata"].pop("run_id", uuid.uuid4().hex),
```

Comment:
When would this occur? Seems odd to invent a cloud resource id.

Reply:
it would occur if that metadata key doesnt exist on the cloud Message, like if the message was inserted manually.

Reply:
Oh I see, `Run.id` already [has a default ](https://github.com/griptape-ai/griptape/blob/d69d2fc47083f2719b7b0f19a4fb3696bfdfd443/griptape/memory/structure/run.py#L16) so should just avoid setting `id` here if `run_id` is not present.